syntax = "proto3";

package messages;

message EmptyRequest {}

message MessageResponse {
  string message = 1;
}

message MessageList {
  repeated string messages = 1;
}

service MessageService {
  rpc GetStaticMessage(EmptyRequest) returns (MessageResponse);
  rpc GetAllMessages(EmptyRequest) returns (MessageList);
}
